### ISSUE TITLE
Feature request: open graph on new window/export graph as image

### ISSUE THREAD

**Issue by Matsu23** [treymo/joplin-link-graph#63]
Seeing large graphs inside joplin is a little messy...

**Comment by KrullTheWarriorKing** [comment]
Not sure this is the good place to say it but **this would be really interesting for my research if I could export my graph**

**Comment by dverbeeck** [comment]
Concur, this would be a super helpful function. Exporting the Link Graph at will is probably preferable to watching/waiting on it to update in real-time.

**Comment by nunesgh** [comment]
This is a must have enhancement to the plugin! I have been essentially leaving the Graph View hidden all the time to save screen space and to avoid refreshing the graph every time I switch notes.

**Comment by zblesk** [comment]
Yes, please. A popup box with the graph, or an entirely separate window, would be best. Otherwise I can't see much of it.
